store/extensions/G-EnterFullRoom/extension.json

```
@@ -28,10 +28,9 @@
       "version": "1.5"
     },
 
-    "language": "Java",
+    "language": "C#",
     "commands": {
-      "default": ["java", "-jar", "GClick.jar", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
-      "linux": ["(optional) <some OS-specific command...>"]
+      "default": [".exe", "-exe", "G-EnterFullRoom.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],

```

This command won't work, use the following:
```suggestion
      "default": ["cmd", "/c", "G-EnterFullRoom.exe", "-c", "{cookie}", "-p", "{port}", "-f", "{filename}"],
```